src/Elliptic/Executables/Punctures/SolvePunctures.cpp

```diff
@@ -0,0 +1,22 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Elliptic/Executables/Punctures/SolvePunctures.hpp"
+
+#include <vector>
+
+#include "Domain/Creators/RegisterDerivedWithCharm.hpp"
+#include "Elliptic/SubdomainPreconditioners/RegisterDerived.hpp"
+#include "Parallel/CharmMain.tpp"
+#include "Utilities/Serialization/RegisterDerivedClassesWithCharm.hpp"
+
+extern "C" void CkRegisterMainModule() {
+  Parallel::charmxx::register_main_module<Metavariables>();
+  Parallel::charmxx::register_init_node_and_proc(
+      {&domain::creators::register_derived_with_charm,
+       &register_derived_classes_with_charm<
+           Metavariables::solver::schwarz_smoother::subdomain_solver>,
+       &elliptic::subdomain_preconditioners::register_derived_with_charm,
+       &register_factory_classes_with_charm<Metavariables>},
+      {});
```

Comment:
why not add the init_proc? I just noticed this on this file, but probably happened in others too

Reply:
The error handling is added by default (see CharmMain.tpp). We dont have any other proc init functions.